cortico.js

```diff
@@ -1500,6 +1535,19 @@ async function setupPreferredPharmacy(code, demographic_no) {
 }
 
 
+async function getDiagnosticFromCortico(appt_no, notes) {
+  const clinicName = localStorage['clinicname']
+  const url = `https://${clinicName}.cortico.ca/api/encrypted/diagnostic-results/?appointment_id=${appt_no}&notes=${notes}`
+
```

Comment:
Please only send the key to Cortico, not the whole notes field. We don't want to transmit any more information over hte network to Cortico than necessary